omero/sysadmins/UpgradeCheck.txt

```diff
@@ -6,7 +6,7 @@ On each startup the OMERO server checks for available upgrades via the
 <components/common/src/ome/system/UpgradeCheck.java>`.
 An HTTP GET call is made to the URL configured in
 :source:`etc/omero.properties` as ``omero.upgrades.url``, currently
-http://upgrade.openmicroscopy.org.uk by default (note that viewing that link
+`http://upgrade.openmicroscopy.org.uk` by default (note that viewing that link
```

Comment:
If you don't want to apply the verbatim markup, an alternative is to prepend the URL with `/`.